packages/react-fetch/src/ReactFetchNode.js

```diff
@@ -154,9 +154,12 @@ Response.prototype = {
       return this._json;
     }
     const buffer = readRecordValue(this._bufferRecord);
-    const json = JSON.parse(buffer.toString());
-    this._json = json;
-    return json;
+    try {
+      this._json = JSON.parse(buffer.toString());
+    } catch (error) {
+      return this.text();
```

Comment:
returning a JSON would be more preferable or a null

Reply:
if you don't mind, you can update the changes, and i will make the commit, i am actually on my ipad and out for some stuff!